packages/testing/src/fixture.ts

```diff
@@ -203,7 +203,7 @@ export class Fixture {
     return output
   }
 
-  async runWithBuildAndIntrospect() {
+  async runWithBuildAndIntrospect(): Promise<Awaited<ReturnType<typeof build>> & { output: string }> {
```

Comment:
While we can totally keep this change in, the typescript error should go away with the other fixes

Reply:
Oh, cool, I was wondering about that!